model_drafts/cards/card.js

```diff
@@ -0,0 +1,11 @@
+var Sequelize = require('sequelize');
+
+function Card(db){
+  var Card = db.define('Card', {
+    type: Sequelize.TEXT,
```

Comment:
I think we could add some additional fields to facilitate in actually moving the player (see pseudocode below). These could be defined here or a reference to another table?

rotation: (0, 90, 180, 270)
direction: boolean (front=true or back=false or neither=null?)
numSpots: integer or null if just rotating